wger/exercises/api/views.py

```diff
@@ -46,6 +48,29 @@
 from wger.utils.permissions import CreateOnlyPermission
 
 
+class ExercisesListSpecial(APIView):
+    @classmethod
+    def get(cls, request):
+        queryset = Exercise.objects.all()
+        serializer = ListAllSerializer(data=queryset, many=True)
+        serializer.is_valid()
+        return Response(serializer.data)
+
+
+class ExerciseSpecial(APIView):
+    @classmethod
```

Comment:
@Manorlds-Eaglespark are you accessing anything on the class in this method? Otherwise, a simple override would suffice.

Reply:
Well, I did override get_queryset method at first and I got context errors because templating is already integrated into this modelviewset.